config/config.txt

```diff
@@ -103,6 +103,9 @@ LOG_WHISPER
 ## log emotes
 LOG_EMOTE
 
+## log subtler emotes
+#LOG_SUBTLER
+
```

Comment:
Could you move those to inside "config/skyrat/skyrat_config.txt" ?

Reply:
Done.